addons/l10n_ch/views/account_invoice_view.xml

```diff
@@ -60,6 +60,12 @@
                         groups="base.group_user"
                         />
                 </xpath>
+                <header position="after">
+                    <field name="l10n_ch_isr_needs_fixing" invisible="1"/>
+                    <div groups="account.group_account_invoice" class="alert alert-warning" role="alert" style="margin-bottom:0px;" attrs="{'invisible': [('l10n_ch_isr_needs_fixing', '=', False)]}">
+                        Please fill in a correct ISR reference in the payment reference.  You will have problems paying later otherwise.
```

Comment:
```suggestion
                        Please fill in a correct ISR reference in the payment reference, or I will personally bust your knee caps
```
I mean, " You will have problems paying later otherwise." is too mysterious (in a bad way). Be more explicit